lib/big_query/base/query.ex

```diff
@@ -21,15 +21,15 @@ defmodule BigQuery.Base.Query do
 
   defp post_params(sql, queryParams) do
     %{
-      "kind": "bigquery#queryRequest",
-      "query": sql,
-      "defaultDataset": %{
-        "datasetId": Env.get(:bq_dataset),
-        "projectId": Env.get(:bq_project_id)
+      kind: "bigquery#queryRequest",
+      query: sql,
+      defaultDataset: %{
+        datasetId: Env.get(:bq_dataset),
+        projectId: Env.get(:bq_project_id)
       },
-      "useLegacySql": false,
-      "parameterMode": "NAMED",
-      "queryParameters": parameterize(queryParams),
+      useLegacySql: false,
+      parameterMode: "NAMED",
+      queryParameters: parameterize(queryParams),
```

Comment:
saving up on those quote chars - it does read more nicely

Reply:
Yeah, Elixir 1.7 complains about unnecessary quotes on these - even though all the docs/examples show them as quoted.